blog-details.html

```diff
@@ -0,0 +1,610 @@
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./public/normalize.css" />
+    <link rel="stylesheet" href="./public/constants.css" />
+    <link rel="stylesheet" href="./public/style.css" />
+  </head>
+  <body>
+    <header class="header list">
+      <a href="./index.html" class="header__logo">
+        <strong class="header__title">
+          <img
+            src="./images/icons/logo.svg"
+            alt="Pixmart logo"
+            width="108"
+            height="24"
+          />
+        </strong>
+      </a>
+      <nav class="header__navigation">
+        <ul class="navigation__list list">
+          <li class="navigation__list-item">
+            <a
+              href="#"
+              class="navigation__link button button__small button__navigation"
+            >
+              About
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./catalog.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Products
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="#"
+              class="navigation__link button button__small button__navigation"
+            >
+              Creators
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./blog-details.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Blog
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./contacts.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Contact
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <div class="header__assets-block list">
+        <form action="#" method="get" class="assets-block__searching">
+          <label for="asset-searching" class="searching__label hidden"
+            >Find asset</label
+          >
+          <input
+            type="text"
+            class="searching__input input input__small"
+            id="asset-searching"
+            placeholder="Search for Digital Assets"
+          />
+          <button type="submit" class="searching__button hidden">Search</button>
+        </form>
+        <a
+          href="#"
+          target="_blank"
+          class="assets-block__button button button__primary button__square-medium"
+        >
+          <svg class="assets-block__icon icon">
+            <use xlink:href="./images/icons/sprite.svg#bag"></use>
+          </svg>
+        </a>
+      </div>
+    </header>
+    <main class="container__main">
+      <section class="blog-introduction container__section list list__column">
```

Comment:
It makes no sense to create the section only for Heading tag(H1).
I think it's better to get rid of this unnecessary Section tag and combine with the following section.
Or replace this Section tag with DIV tag

Reply:
Fixed 🥇